### DISCUSSION TITLE
[Question]: What should I do if the tsx file has no style hint?

### DISCUSSION THREAD

**Discussion by Kevin-free** [danny-avila/LibreChat#2751] [Q&A] [answered]
### What is your question?

Why is there no prompt when writing style in tsx file in VSCode?
The project has not been modified. There is `tailwind.config.cjs` in the `client` directory.

<img width="482" alt="image" src="https://github.com/danny-avila/LibreChat/assets/26422757/ae25ccba-2965-4650-94c5-4968823ad727">

### More Details

Executed `npm install -D tailwindcss`.

In addition, the `tsconfig.json` file is marked with a red prompt. Does this have any impact and how to deal with it?

<img width="570" alt="image" src="https://github.com/danny-avila/LibreChat/assets/26422757/96c64b76-5a1f-45a5-affa-a91131b8dd5a">

### What is the main subject of your question?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I'm not sure what you mean. Is there some IDE intellisense you are expecting for tailwindcss?

**Reply by Kevin-free** [reply]
I have installed the plug-in Tailwind CSS IntelliSense in VSCode, and I hope that smart completion prompts will pop up when writing tailwindCSS code. For example, entering `className="tex"` should automatically prompt `text-...` to wait for the option.

**Strangely it works now**, thanks! Sorry to bother you!

**Reply by danny-avila** [reply, marked as answer]
Any time I have typescript issues, I restart the TS server. You can find this in the command palette.

I might look into this plugin, thanks for the tip!
